### DISCUSSION TITLE
Format disks/partitions when installing OS

### DISCUSSION THREAD

**Discussion by shipler** [Security-Onion-Solutions/securityonion#1579] [Unsupported Versions] [answered]
When installing on physical server with multiple disks (sda for os and sdb for data) the installer gets an error that partitions with the same names already exist. 
To solve this issue it requires full format of partitions (using live image) and only than it's possible to install SO.

Maybe you can add an option for - would you like to destroy current partitioning of the disks...

**Comment by dougburks** [comment, marked as answer]
@TOoSmOotH created https://github.com/Security-Onion-Solutions/securityonion/issues/1603 for this.
